packages/desktop-client/src/components/reports/SaveReport.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+
+import ExpandArrow from '../../icons/v0/ExpandArrow';
+import Button from '../common/Button';
+import Menu from '../common/Menu';
+import MenuTooltip from '../common/MenuTooltip';
+import Text from '../common/Text';
+import View from '../common/View';
+
+function onMenuSelect(item, setMenuOpen) {
+  switch (item) {
+    case 'save':
+      setMenuOpen(false);
+      break;
+    case 'clear':
+      setMenuOpen(false);
+      break;
+    default:
+  }
+}
+
+function SaveReportMenu({ setMenuOpen }) {
+  return (
+    <MenuTooltip width={150} onClose={() => setMenuOpen(false)}>
+      <Menu
+        onMenuSelect={item => {
+          onMenuSelect(item, setMenuOpen);
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏since the `onMenuSelect` util is used only 1x - it would be nicer to just inline the code here. Makes it easier to read it without having to scroll up and down.